nautobot/extras/constants.py

```diff
@@ -30,6 +30,7 @@
     "soft_time_limit",
     "time_limit",
     "has_sensitive_variables",
+    "is_singleton",
```

Comment:
Do we need to add `nautobot_job_ignore_singleton_lock` to `JOB_RESULT_CUSTOM_CELERY_KWARGS` below?

Reply:
See [this](https://github.com/nautobot/nautobot/pull/6431/files/1ff1a2ea39568fb057923c2b2f2e9d76df88b071#r1849663502) comment.

Reply:
Added as per discussion.